libs/langgraph/langgraph/graph/state.py

```diff
@@ -202,7 +218,7 @@ def __init__(
                 stacklevel=2,
             )
             if output_schema is None:
-                output_schema = cast(Union[type[OutputT], None], output)
+                output_schema = cast(type[OutputT], output)
```

Comment:
Sure this change is right? Output and input schemas are not required args, so they can have the value of None no?